src/app/icd/server/ICDManager.h

```diff
@@ -66,7 +74,7 @@ class ICDManager : public ICDListener
     };
 
     ICDManager() {}
-    void Init(PersistentStorageDelegate * storage, FabricTable * fabricTable, Crypto::SymmetricKeystore * symmetricKeyStore,
+    void Init(PersistentStorageDelegate * storage, FabricTable * fabricTable, Crypto::SessionKeystore * symmetricKeyStore,
```

Comment:
isn't there a impact in changing this to the `SessionKeyStore`? I know it is a derivative, but I believe there was a reason for it. @mkardous-silabs

Reply:
The reason is similar to the comment below. This type is defined in the `ICDMonitoringTable.h`, which should be excluded from build for SITs, as it's pretty heavy.

Reply:
It was mostly for clarify since we are not using "Session" keys. From compiler perspective, we can define the SymmetricKeystore alias in the ICDManager and the ICDMonitoringTable.

Do you think this would be acceptable?
@jmartinez-silabs

Reply:
It seems reasonable to me, I pushed an update containing suggested changes.